src/OctopusPuppet.Gui/Views/NavigationService.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Diagnostics;
+using System.Text.RegularExpressions;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Documents;
+
+namespace OctopusPuppet.Gui.Views
+{
+    public static class NavigationService
+    {
+        // Copied from http://geekswithblogs.net/casualjim/archive/2005/12/01/61722.aspx
```

Comment:
Is this still true?

Reply:
We changed the regex, the rest is the same